modules/ballerina-core/src/main/java/org/ballerinalang/model/util/MessageUtils.java

```diff
@@ -75,7 +75,7 @@ public static String getStringFromInputStream(InputStream in) {
 //            Long value = (Long) orgBValue.getValue();
 //            bValue = new LongValue(value);
 //        } else if (orgBValue instanceof FloatValue) {
-//            Float value = (Float) orgBValue.getValue();
+//            floattype value = (floattype) orgBValue.getValue();
```

Comment:
Whey did we rename float to floattype

Reply:
This is because float was changed to floattype since folat is a java keyword. This was fixed with the last commit